terraform/variables/prod-us.tfvars

```diff
@@ -69,6 +68,12 @@ ingestors = {
     }
   }
 }
+cluster_settings = {
+  initial_node_count : 4
+  min_node_count : 4
+  max_node_count : 5
```

Comment:
The only reason that `initial_node_count` and `min_node_count` were 4 was to hack around a GKE-imposed resource limit on the number of Kubernetes jobs per namespace, which was necessary back when we spawned one k8s job per task. We don't need to do that anymore, so we can use lower values to allow the cluster to scale down if nodes are idle. Let's try `initial_node_count = 1` and `min_node_count = 3`.